package.json

```diff
@@ -153,6 +154,11 @@
         "command": "csharp.listProcess",
         "title": "List process for attach",
         "category": "CSharp"
+      },
+      {
+        "command" : "csharp.listRemoteProcess",
+        "title" : "List processes on remote connection for attach",
+        "category": "CSharp"
```

Comment:
I don't know what `category` is supposed to be for. You can see if the schema will tell you. @chuckries created the downladDebugger command, so he might know. Otherwise you can look at the VS Code source to see if that has the answer (http://github.com/Microsoft/VSCode)